ksqldb-engine/src/main/java/io/confluent/ksql/physical/scalablepush/ScalablePushRegistry.java

```diff
@@ -52,6 +52,7 @@
 
   private final PushLocator pushLocator;
   private final LogicalSchema logicalSchema;
+  private final boolean isTable;
```

Comment:
This is not for this line but while reviewing this class I found that in https://github.com/confluentinc/ksql/pull/7661/files#diff-350c1eaff8254e08b2c4db5fab86e15d71b9e6a1590a698e2adc0e89a1e7e1c2R161 below, within

```
    public void process(final Object key, final GenericRow value) {
```

we do not need to call `this.context.forward(key, value);` after `handleRow` since this should never have any children processors, and if it ever do, calling it may result in unwanted behavior.

Reply:
Ok, makes sense.  I didn't know if it was considered good practice to do this (even if you know there are no children). I removed that line.